src/utils.c

```diff
@@ -78,15 +80,26 @@ void utils_open_browser(const gchar *uri)
 	g_return_if_fail(uri != NULL);
 	win32_open_browser(uri);
 #else
-	gchar *argv[2] = { (gchar *) uri, NULL };
+	gchar *new_cmd, *argv[2] = { (gchar *) uri, NULL };
 
 	g_return_if_fail(uri != NULL);
 
-	while (!spawn_async(NULL, tool_prefs.browser_cmd, argv, NULL, NULL, NULL))
+	while (1)
 	{
-		gchar *new_cmd = dialogs_show_input(_("Select Browser"), GTK_WINDOW(main_widgets.window),
-			_("Failed to spawn the configured browser command. "
-			  "Please correct it or enter another one."),
+		/* Uses the user's default browser akin to xdg-open (in flatpak through a portal) */
+		if (EMPTY(tool_prefs.browser_cmd))
+		{
+			if (gtk_show_uri_on_window(GTK_WINDOW(main_widgets.window), uri, GDK_CURRENT_TIME, NULL))
```

Comment:
If we do care about pre-3.22 GTK, it'd be easy enough to do something like that:
```c
#if GTK_CHECK_VERSION(3, 22, 0)
			if (gtk_show_uri_on_window(GTK_WINDOW(main_widgets.window), uri, GDK_CURRENT_TIME, NULL))
#else
			if (gtk_show_uri(gtk_widget_get_screen(main_widgets.window), uri, GDK_CURRENT_TIME, NULL))
#endif
```

Reply:
But we don't